typescript/infra/config/environments/mainnet3/governance/timelock/aw.ts

```diff
@@ -0,0 +1,4 @@
+import { ChainMap } from '@hyperlane-xyz/sdk';
+import { Address } from '@hyperlane-xyz/utils';
+
+export const awTimelocks: ChainMap<Address> = {};
```

Comment:
I assume both this and the regular timelocks will be populated after the prod deployment

Reply:
yup! wanted to do that separately once we finalise the min delay